Hmm, that looks real:
```
===== 620 passed, 21 skipped, 2 xpassed, 287 warnings in 246.81s (0:04:06) =====
+ [[ 1 == \1 ]]
+ make mypy
mypy galaxy tests
galaxy/jobs/__init__.py:957: error: "MinimalManagerApp" has no attribute
"job_config"  [attr-defined]
    ...Mapper(self, queue.dispatcher.url_to_destination, self.app.job_config)
                                                         ^
galaxy/jobs/handler.py:189: error: "MinimalManagerApp" has no attribute
"job_config"  [attr-defined]
    ...mGrabber.get_grabbable_handler_assignment_method(self.app.job_config.h...
                                                        ^
galaxy/jobs/handler.py:195: error: "MinimalManagerApp" has no attribute
"job_config"  [attr-defined]
                    max_grab=self.app.job_config.handler_max_grab,
                             ^
galaxy/jobs/handler.py:196: error: "MinimalManagerApp" has no attribute
"job_config"  [attr-defined]
                    self_handler_tags=self.app.job_config.self_handler_tag...
                                      ^
galaxy/jobs/handler.py:197: error: "MinimalManagerApp" has no attribute
"job_config"  [attr-defined]
                    handler_tags=self.app.job_config.handler_tags,
                                 ^
Found 5 errors in 2 files (checked 483 source files)
make: *** [Makefile:96: mypy] Error 1
```

--- a/lib/galaxy/structured_app.py
+++ b/lib/galaxy/structured_app.py
@@ -66,6 +66,7 @@ class MinimalManagerApp(MinimalApp):
     role_manager: Any  # 'galaxy.managers.roles.RoleManager'
     installed_repository_manager: Any  # 'galaxy.tool_shed.galaxy_install.installed_repository_manager.InstalledRepositoryManager'
     user_manager: Any
+    job_config: Any  # 'galaxy.jobs.JobConfiguration'
     job_manager: Any  # galaxy.jobs.manager.JobManager
 
     @property
@@ -113,13 +114,13 @@ class StructuredApp(MinimalManagerApp):
     tool_data_tables: Any  # 'galaxy.tools.data.ToolDataTableManager'
     genomes: Any  # 'galaxy.visualization.genomes.Genomes'
     error_reports: Any  # 'galaxy.tools.error_reports.ErrorReports'
-    job_config: Any  # 'galaxy.jobs.JobConfiguration'
     tool_cache: Any  # 'galaxy.tools.cache.ToolCache'
     tool_shed_repository_cache: Any  # 'galaxy.tools.cache.ToolShedRepositoryCache'
     watchers: Any  # 'galaxy.config_watchers.ConfigWatchers'
     installed_repository_manager: Any  # 'galaxy.tool_shed.galaxy_install.installed_repository_manager.InstalledRepositoryManager'
     workflow_scheduling_manager: Any  # 'galaxy.workflow.scheduling_manager.WorkflowSchedulingManager'
     interactivetool_manager: Any
+    job_config: Any  # 'galaxy.jobs.JobConfiguration'
     job_manager: Any  # galaxy.jobs.manager.JobManager
     user_manager: Any
     api_keys_manager: Any  # 'galaxy.managers.api_keys.ApiKeyManager'

